proto/http.c

```diff
@@ -235,6 +236,7 @@ static int http_parse(struct wsgi_request *wsgi_req, char *watermark) {
 
 	// REQUEST_URI / PATH_INFO / QUERY_STRING
 	base = ptr;
+	QS:
```

Comment:
labels on column 0 please, also something more descriptive than QS may be helpful